### ISSUE TITLE
Typebot viewer is not working as expected

### ISSUE THREAD

**Issue by matheusbruno97** [baptisteArno/typebot.io#928]
Hi there,

First of all, thank you so much for this project. Typebot is simply AMAZING! As soon as I learn more about development, I will definitely contribute to this open-source project.

I wanted to ask you about the viewer image. I tried using the typebot-viewer:latest image and it simply doesn't work. I always get a 500 server error as response. While I was trying to troubleshoot this, I could discover that if I change the image to version 2.16 the viewer _kind of_ works.

I say _kind of works_ because I do not receive a 500 error page. However, the text bubbles seem to disappear. So it only shows gifs and options.

TL;DR = latest version doesn't work and version 2.16 works with a few issues.

To illustrate this, I've made a short video explaining what is happening. On the middle of it, I edit the docker-compose.yml file so the viewer image is changed to version 2.16.

Please let me know if I'm doing anything wrong. Looking forward to your reply!

Here is the video: https://www.loom.com/share/fb271790d2b04e5abfcbb055535d6b1c?sid=b9f04e50-e7c0-432b-982e-09a6eb1de64b

Kind regards,
Bruno

**Comment by baptisteArno** [comment]
Hi Bruno,

Can you provide the server logs? You should see a clear error here.

`docker-compose logs`

**Comment by matheusbruno97** [comment]
Hello!

Thank you for your prompt reply. I saw many errors there and chatGPT helped me debug this. It was missing the NEXTAUTH_URL variable in the viewer.

I appreciate your insight on the server logs. I'm new to this and I didn't know we could see logs from Docker.

Thanks anyway!

PS: I'm attaching the log file, just in case you want to check it out.
[logs.md](https://github.com/baptisteArno/typebot.io/files/12917762/logs.md)

**Comment by seiler-emerson** [comment]
@matheusbruno97 managed to solve?
I have the same problem installing on my own server with Ubuntu 22.04.3.
Error on click button view: Error! Could not reach server. Check your connection.{}

Error in my typebot-viewer container:
` ⨯ Error: Invalid environment variables: {"NEXTAUTH_URL":["Required"]}
    at onValidationError (/app/apps/viewer/.next/server/chunks/833.js:1:9209)
    at r (/app/apps/viewer/.next/server/chunks/833.js:1:13281)
    at r (/app/apps/viewer/.next/server/chunks/833.js:1:13792)
    at /app/apps/viewer/.next/server/chunks/833.js:1:8538` 

However, I define the NEXTAUTH variable in my compose correctly:

`      - NEXTAUTH_URL=http://typebot1.mywebsite.dev
      - NEXT_PUBLIC_VIEWER_URL=https://typebot2.mywebsite.dev
      - NEXTAUTH_URL_INTERNAL=http://localhost:3000`
      
    I'm publishing the site through cloudflare tunnel, could it be related?
